src/currency-field/CurrencyField.ts

```diff
@@ -166,17 +211,27 @@ export class CurrencyField extends OmniFormElement {
         }
     }
 
+    // Parse the cents portion of the currency value.
     _parseFraction(value: string): string {
         let cleanValue = '';
+
         for (let i = 0; i < value.length; i++) {
             const character = value.charAt(i);
             if (/\d/.test(character)) {
                 cleanValue += character;
             }
         }
+
         return cleanValue;
     }
 
+    // Blur when the enter key is pressed on a virtual keyboard.
+    _blurOnEnter(e: any) {
```

Comment:
Why is `e` an `any` and not `KeyboardEvent`?

Reply:
Updated to KeyboardEvent implemented the same on components that also use this functionality.